electricitymap/contrib/capacity_parsers/FI.py

```diff
@@ -0,0 +1,62 @@
+from datetime import datetime, timedelta
+from typing import Any
+
+from requests import Response, Session
+
+from electricitymap.contrib.capacity_parsers.ENTSOE import (
+    fetch_production_capacity as fetch_entsoe_production_capacity,
+)
```

Comment:
I'm a little concerned about importing other parsers without having some form of testing in place to ensure we don't break other parsers. 

For example I have https://github.com/electricitymaps/electricitymaps-contrib/pull/6496 open which could change how things are working in here.